packages/cluster-ui/babel.config.js

```diff
@@ -17,7 +17,8 @@ const presets = [
 const plugins = [
   "@babel/proposal-class-properties",
   "@babel/proposal-object-rest-spread",
-  ["import", { "libraryName": "antd", "style": "css" }]
+  "@babel/plugin-transform-runtime",
```

Comment:
Can you add a comment here to explain that this plugin was added to support dynamic loading into CC